src/Image/image.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { mount, render } from 'enzyme';
+import { block as Image, ImageProps } from './index';
+import { defaultConfig } from './configs';
+
+describe('Image component', () => {
+  let props: ImageProps;
+
+  beforeEach(() => {
+    props = {
+      ...defaultConfig,
+      globalSettings: {},
+      utils: {
+        isAmpRequest: false
+      }
+    };
+  });
+
+  it('matches snapshot', () => {
+    const rendered = render(<Image {...props} />);
+    expect(rendered).toMatchSnapshot();
+  });
+
+  it('should have an img tag when a normal request', () => {
+    const image = mount(<Image {...props} />);
+    expect(image.find('img').exists()).toBeTruthy();
+  });
+
+  it('should have an amp-img tag when an AMP request', () => {
+    props.utils.isAmpRequest = true;
+    const image = mount(<Image {...props} />);
+    expect(image.find('img').exists()).toBeFalsy();
+    expect(image.find('amp-img').exists()).toBeTruthy();
```

Comment:
Since the `amp-img` version has a different HTML structure, let's add a snapshot test here too.